From c338573929657e42092f09f05d84b9db0595c3f3 Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Tue, 22 Jul 2014 14:36:37 -0400
Subject: [virt] kvm/ppc: reserve a capability number for multitce support

Message-id: <1406040016-3289-3-git-send-email-pbonzini@redhat.com>
Patchwork-id: 86081
O-Subject: [RHEL7 PATCH v2 002/221] KVM: PPC: reserve a capability number for multitce support
Bugzilla: 1116936
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Alexey Kardashevskiy <aik@ozlabs.ru>

This is to reserve a capablity number for upcoming support
of H_PUT_TCE_INDIRECT and H_STUFF_TCE pseries hypercalls
which support mulptiple DMA map/unmap operations per one call.

Signed-off-by: Alexey Kardashevskiy <aik@ozlabs.ru>
Signed-off-by: Gleb Natapov <gleb@redhat.com>

Conflict:
 include/uapi/linux/kvm.h [93 not in RHEL, 96 and 97 already there]
(cherry picked from commit 0bd50dc971aad3c29043de4fb7bce45c351d1b67)
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/uapi/linux/kvm.h b/include/uapi/linux/kvm.h
index 0434c15..b7ebf28 100644
--- a/include/uapi/linux/kvm.h
+++ b/include/uapi/linux/kvm.h
@@ -666,6 +666,7 @@ struct kvm_ppc_smmu_info {
 #define KVM_CAP_IRQ_MPIC 90
 #define KVM_CAP_PPC_RTAS 91
 #define KVM_CAP_IRQ_XICS 92
+#define KVM_CAP_SPAPR_MULTITCE 94
 #define KVM_CAP_HYPERV_TIME 96
 #define KVM_CAP_IOAPIC_POLARITY_IGNORED 97
 
-- 
1.7.1